Modules/Office365DSC/Modules/Office365DSCUtil.psm1

```diff
@@ -1450,8 +1450,8 @@ function Test-SPOServiceConnection
     )
     $VerbosePreference = 'SilentlyContinue'
     $WarningPreference = "SilentlyContinue"
-    Write-Verbose -Message "Verifying the LCM connection state to SharePoint Online"
-    $catch = Connect-SPOService -Url $SPOCentralAdminUrl -Credential $GlobalAdminAccount
+    Write-Verbose "Verifying the LCM connection state to SharePoint Online"
```

Comment:
We recently standardized all the Verbose output. Please us -Message

Reply:
Done.